qcengine/programs/molpro.py

```diff
@@ -151,12 +160,15 @@ def build_input(self, input_model: 'ResultInput', config: 'JobConfig',
         if template is None:
             input_file = []
 
+            # Resolving keywords
+            caseless_keywords = {k.lower(): v for k, v in input_model.keywords.items()}
+            unrestricted = False
+            if "reference" in caseless_keywords and caseless_keywords["reference"] == "unrestricted":
```

Comment:
Did you want to autodetect based on multiplicity like the Psi4 driver?

Reply:
No I was thinking to follow the Molpro default which `ROHF` is done by default for open-shell calculations with only `UHF` performed if requested.

Reply:
Ah gotcha, can we add a comment here to this effect?

Reply:
Yes definitely.